web/app/scripts/social-costs/social-costs-directive.js

```diff
@@ -5,11 +5,35 @@
     function SocialCosts() {
         var module = {
             restrict: 'EA',
-            templateUrl: 'scripts/social-costs/social-costs.html',
+            templateUrl: function(elem, attrs) {
+                // Note that use of templateUrls is discouraged in most cases
```

Comment:
Since they're used in different places and not set dynamically, what about just defining two directives, e.g. `driver-social-costs` and `driver-social-costs-tool`?